Base/QTGUI/qSlicerModulesMenu.cxx

```diff
@@ -556,6 +562,11 @@ void qSlicerModulesMenu::addModule(qSlicerAbstractCoreModule* moduleToAdd)
   foreach(const QString& category, module->categories())
     {
     QMenu* menu = d->menu(this, category.split('.'), module->isBuiltIn());
+    if (!menu)
+      {
+      qWarning() << "Failed to add module" << module->name() << "to the menu" << category << "because this string is already used as module name.";
```

Comment:
```suggestion
      qWarning() << "Failed to add module " << module->name() << " to the menu " << category << " because this string is already used as module name.";
```

Reply:
`qWarning` streaming operator adds those spaces automatically (unless some special directive is specified).

I find this behavior unintuitive, too. I see that it can make things a little more convenient, but it also makes some things more complicated (e.g., if you want to use your own separators or you want to put quotes around strings).

Reply:
Interesting, okay, I didn't know that. 👍   I searched for qWarning in the Slicer source code and saw we typically use spaces so I thought we should be consistent.  Looking at it now I think we probably shouldn't rely on that feature since it's a special case that could lead to error in other kinds of messages.

Reply:
How can i use these codes please

Reply:
You don't need to do anything with this code. The fix is integrated and available in the latest Slicer version.

Reply:
I don't know anything about code at all, but I really need this to fix my 3Dslicer，I would really appreciate it if you could help me fix my 3Dslicer。o(╥﹏╥)o

Reply:
It is already fixed in the latest Slicer Preview Release. It is avaliable for download.

It would be nice if someone would also fix the Chinese translation on [Weblate](https://hosted.weblate.org/translate/3d-slicer/3d-slicer/zh_Hans/?q=&sort_by=-priority%2Cposition&checksum=) by translating "Segmentation" (module category) and "Segmentations" (module name) words with two different words, because the crash is due to that they both translated as 分割

Reply:
I understand. I downloaded the Stable Release of the software. I'll try reinstalling a Preview Release now. Thank you very much!

Reply:
I did it! Thank you very much!